composer.json

```diff
@@ -9,7 +9,13 @@
     "ext-dom": "*",
     "ext-json": "*",
     "ext-libxml": "*",
-    "ext-mbstring": "*"
+    "ext-mbstring": "*",
+    "ampproject/amp-wp": "dev-develop",
```

Comment:
I assume this should be changed to `^2.0.2` soon?

Reply:
Yeah, once implemented we probably want to stick to stable versions.